ops/jujucontext.py

```diff
@@ -50,9 +98,6 @@ class _JujuContext:
     """The root directory of the charm where it is running.
```

Comment:
We use this description for `Framework.charm_dir` - could use the same?

```suggestion
    """The charm project root directory.
```